### DISCUSSION TITLE
How can I delete a view in memory

### DISCUSSION THREAD

**Discussion by mauricioLopezRamirez** [adonisjs/core#2211] [Help] [answered]
I want to delete a template in memory after having used it.

I can't find how to remove the template from memory in ->  [https://preview.adonisjs.com/guides/views/in-memory-views](https://preview.adonisjs.com/guides/views/in-memory-views)

**Comment by thetutlage** [comment, marked as answer]
Yup, currently there is no API for that. But I can add it in the next release
